src/component/shared/PageSearch.jsx

```diff
@@ -0,0 +1,225 @@
+import React from 'react'
+import Downshift from 'downshift'
+import { makeStyles } from '@material-ui/core/styles'
+import {
+    TextField,
+    InputAdornment,
+    Paper,
+    Menu,
+    MenuItem,
+    Button,
+    withStyles,
+    ClickAwayListener,
+    Box,
+} from '@material-ui/core'
+import SearchIcon from '@material-ui/icons/Search'
+import relayerUrl from '../../asset/relayer.svg'
+import issuerUrl from '../../asset/issuer.svg'
+import contractUrl from '../../asset/contract.svg'
+
+const Icons = {
+  'relayer': relayerUrl,
+  'token': issuerUrl,
+  'contract': contractUrl,
+}
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  container: {
+    flexGrow: 1,
+    position: 'relative',
+  },
+  paper: {
+    position: 'absolute',
+    zIndex: 1,
+    marginTop: theme.spacing(1),
+    left: 0,
+    right: 0,
+    [theme.breakpoints.down('sm')]: {
+      position: 'unset',
+    },
+  },
+  inputRoot: {
+    flexWrap: 'wrap',
+    [theme.breakpoints.down('sm')]: {
+      backgroundColor: '#323252',
+    },
+  },
+  inputInput: {
+    width: 'auto',
+    flexGrow: 1,
+  },
+}))
+
+const renderInput = (inputProps) => {
+  const { InputProps, classes, ref, ...other } = inputProps
+
+  return (
+    <TextField
+      InputProps={{
+        inputRef: ref,
+        classes: {
+          root: classes.inputRoot,
+          input: classes.inputInput,
+        },
+        endAdornment: (
+          <InputAdornment position="end">
+            <SearchIcon />
+          </InputAdornment>
+        ),
+        ...InputProps,
+      }}
+      {...other}
+    />
+  )
+}
+
+const renderSuggestion = (suggestionProps) => {
+  const { suggestion, index, itemProps, highlightedIndex, selectedItem } = suggestionProps
+  const isHighlighted = highlightedIndex === index
+  const isSelected = (selectedItem || '').indexOf(suggestion.label) > -1
+
+  return (
+    <MenuItem
+      {...itemProps}
+      key={suggestion._id}
+      selected={isHighlighted}
+      component="div"
+      style={{
+        fontWeight: isSelected ? 500 : 400,
+      }}
+    >
+      <img style={{width: 15, height: 15, marginRight: 7}} src={Icons[suggestion.type]} alt={suggestion.type}/>
```

Comment:
- spacing at closing tag
- when too manny inline-style, switch to use separate objects